### ISSUE TITLE
Install/restore of a particular package fails with "Package contains multiple nuspec files."

### ISSUE THREAD

**Issue by jmyersmsft** [NuGet/Home#3231]
Steps to repro:
1. Use 3.5.0-rc build 1657. 3.5.0-beta2 does not have the bug.
2. `nuget locals -Clear All`
3. `nuget install Microsoft.TeamFoundationServer.Client -Version 14.83.0` (specifically this version of this package)

This also occurs at least when restoring a packages.config with this specific package version. I manually downloaded the package and it only appears to have one nuspec. Strangely, running the same install command again without clearing the caches does _not_ repro the bug.

**Comment by rrelyea** [comment]
@jmyersmsft  - Can you please try a "dev branch", not "release-3.5.0-rc" build? http://nuget.org/downloads will have it at the bottom of page. let us know what build you try. thx.

**Comment by jmyersmsft** [comment]
@rrelyea same behavior with 3.5.0-rc1-1667 which is the build currently available from the nightly builds link

**Comment by jmyersmsft** [comment]
I believe I've found the issue.

Microsoft.TeamFoundationServer.Client 14.83.0 does only contain one nuspec. However, it is named Microsoft.TeamFoundation.Client.nuspec (note the lack of the word Server). When the package is unpacked, Microsoft.TeamFoundation.Client.nuspec ends up on disk in the global packages folder, but so does a copy of it named microsoft.teamfoundationserver.client.nuspec, so there are then two .nuspec files in the folder.

I verified that this was the critical element by modifying the package so that the name of the nuspec matched the package name, pushing the modified package to a private package source, clearing my locals, and installing again from the private source. The package installed without complaint.
